edx_proctoring/api.py

```diff
@@ -1426,7 +1426,7 @@ def update_attempt_status(attempt_id, to_status,
 
     all_attempts = get_user_attempts_by_exam_id(user_id, exam_id)
 
-    if can_update_credit_grades_and_email(all_attempts, to_status):
+    if can_update_credit_grades_and_email(all_attempts, to_status) or exam['is_practice_exam']:
```

Comment:
This is a bit obscure. I'd rather we change the interface to `can_update_credit_grades_and_email` to reflect that the question it answers requires more information now